docs/import/manage_layers.js

```diff
@@ -11,7 +11,6 @@ export {
   createAssetPickers,
   createLayerRow,
   createOptionFrom,
```

Comment:
Not new, but createAssetPickers isn't actually imported anywhere (just checked on master) so could remove